### ISSUE TITLE
Smart cast breaks hands

### ISSUE THREAD

**Issue by iSuperCoolMan** [ValveSoftware/Dota2-Gameplay#13374]
### Description

Hello! I encountered a problem using smart quick cast in the game. I put a normal application on the buttons, and a quick cast on the same buttons with the alt modifier. But it turned out that when using the keyboard this way, the hand was in a very unnatural position. I have health problems that force me to take care of the position of my hands.

I suggest add to the game the opportunity to make any possible buttons a modifier for buttons, or, for example, a separate setting that will allow you to select one or several buttons that cannot be used in any other way. Thank you for your attention!

---

Здравствуйте! Столкнулся с проблемой использования умного быстрого применения (quick cast) в игре. Поставил обычное применение на кнопки, и быстрое применение на те же кнопки с модификатором альт. Но оказалось что рука при таком использовании клавиатуры находится в очень неестественном положении. Лично у меня есть проблемы со здоровьем, которые вынуждают меня заботиться о положении своих рук.

Предлагаю в игру добавить возможность делать модификатором для кнопок любые возможные кнопки, или например отдельную настройку, которая позволит выбирать одну, или несколько кнопок, которые нельзя будет использовать никаким другим образом. Спасибо за внимание!

### Screenshots

![рука 1](https://github.com/ValveSoftware/Dota2-Gameplay/assets/101731621/08ebafb7-1c6d-429d-87f9-3a90bd4fab37)
![рука 2](https://github.com/ValveSoftware/Dota2-Gameplay/assets/101731621/95fee743-13dc-47cf-a8ca-9748b0f655d8)
![Без имени](https://github.com/ValveSoftware/Dota2-Gameplay/assets/101731621/d51323e6-3b46-4ead-af86-1e45d12b9faa)

**Comment by Aluerie** [comment]
there is already one. I was blinking just fine with pressing N+C behind this screenshot. 
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/33165440/cae0e622-2efe-46b7-ac31-074bca51e6de)

**Comment by commentator24** [comment]
not sure if this helps, but to press ctrl / alt, I use 5th metacarpal head / proximal interphalangeal (PIP) joint instead of thumb. 
Takes a bit of practice but I think easier on the hand once used to it. you can also disable windows key if needed.
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/109324155/477c85e2-0cd9-491c-bada-7ab75922a7e9)
let me know this works for you 😄

I also changed a few hotkeys that are easier to remember:
Q for quickbuy (useful to quickbuy before you die, so you don't lose unreliable gold)
skill 1 = T, ult G. 
skill 4 5 = F R
D for dagon / anything cast on enemy
B for blink
Z for tp

**Comment by iSuperCoolMan** [comment]
> there is already one. I was blinking just fine with pressing N+C behind this screenshot. ![image](https://user-images.githubusercontent.com/33165440/280979297-cae0e622-2efe-46b7-ac31-074bca51e6de.png)

No, this option does not change alt as the only key that can be assigned simultaneously with another for any action. You can press any keys at the same time, but as far as I understand, only 2 keys can be assigned to one action, one of which can only be alt. And this, in my opinion, is strange; it is worth either removing this option completely, or making the settings more flexible.

**Comment by iSuperCoolMan** [comment]
> not sure if this helps, but to press ctrl / alt, I use 5th metacarpal head / proximal interphalangeal (PIP) joint instead of thumb. Takes a bit of practice but I think easier on the hand once used to it. you can also disable windows key if needed. ![image](https://user-images.githubusercontent.com/109324155/281045036-477c85e2-0cd9-491c-bada-7ab75922a7e9.png) let me know this works for you 😄
> 
> I also changed a few hotkeys that are easier to remember: Q for quickbuy (useful to quickbuy before you die, so you don't lose unreliable gold) skill 1 = T, ult G. skill 4 5 = F R D for dagon / anything cast on enemy B for blink Z for tp

Pressing ctrl this way would be really comfortable for me, thanks for the advice, but pressing alt unfortunately puts my hand in an even more uncomfortable position.

**Comment by Aluerie** [comment]
> No, this option does not change alt as the only key that can be assigned simultaneously with another for any action.

I just said that I was using Blink Dagger with N+C combination (without alt) in that demo mode. The UI still says "alt" but you can press alternative modifier.

**Comment by iSuperCoolMan** [comment]
> > No, this option does not change alt as the only key that can be assigned simultaneously with another for any action.
> 
> I just said that I was using Blink Dagger with N+C combination (without alt) in that demo mode. How is it not what you want? The UI still says "alt" but you can press alternative modifier.

Yes, it really works, it’s strange that in the settings you still need to assign keys with alt. I think the problem still requires attention, although this will be enough for me. Thank you!
